src/blockdata/script.rs

```diff
@@ -433,42 +439,42 @@ impl Script {
     }
 
     /// Write the assembly decoding of the script to the formatter.
-    pub fn fmt_asm(&self, f: &mut dyn fmt::Write) -> fmt::Result {
+    pub fn fmt_asm(script: &[u8], f: &mut dyn fmt::Write) -> fmt::Result {
```

Comment:
Is that worth the effort, you think? The fmt asm method is really already a helper that I don't think has any real users. I can wait until we do a breaking release for this to be included.

Reply:
As it is this is a breaking change as far as I can tell since `fmt_asm` is public. I think this little change could make this PR non-breaking, allowing us to change the milestone to 0.26.1. Afaik we wanted to get away from frequent API breaks.

Reply:
ack